x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/sample_logs_input.tsx

```diff
@@ -141,29 +150,39 @@ export const SampleLogsInput = React.memo<SampleLogsInputProps>(({ integrationSe
   const onChangeLogsSample = useCallback(
     (files: FileList | null) => {
       const logsSampleFile = files?.[0];
+
+      setSampleFileError(undefined);
+      setIntegrationSettings({
+        ...integrationSettings,
+        logsSampleParsed: undefined,
+        samplesFormat: undefined,
+      });
```

Comment:
Why is this part move out?

Reply:
I've added a screenshot of the previous behavior to the description as well.